packages/changelog-linker/src/Github/GithubApi.php

```diff
@@ -98,10 +103,9 @@ public function isPullRequestMergedToBaseBranch(int $pullRequestId, string $base
      */
     private function getPullRequestsSinceId(int $id, ?string $baseBranch = null): array
     {
-        $maxPage = 10; // max. 1000 merge requests to dump
-
+        // max. 1000 merge requests to dump
```

Comment:
this comment should have been moved to the newly introduced property

Reply:
That's how php-parser handles comments. It's always the next node
 

I've noticed it and forgot, fixed now. Thanks for reminder :+1: